client/dashboard/domains/name-servers/form.tsx

```diff
@@ -0,0 +1,219 @@
+// eslint-disable-next-line no-restricted-imports
+import { CHANGE_NAME_SERVERS_FINDING_OUT_NEW_NS } from '@automattic/urls';
```

Comment:
If this is not allowed, how will we handle the URLs within the dashboard?

Reply:
We add to `.eslintrc` like this 😄 https://github.com/Automattic/wp-calypso/pull/104914/files#diff-8334e653ee6c11d7c976496bd733faaeee90d6857aa8a3bcce122bd59a6f802dR49

Reply:
hehehe :) Then, I'll keep this approach. 👌